twake/backend/node/src/services/global-resolver.ts

```diff
@@ -0,0 +1,153 @@
+import { CompanyServiceImpl } from "./user/services/companies";
+import { RealtimeServiceAPI } from "../core/platform/services/realtime/api";
+import WebServerAPI from "../core/platform/services/webserver/provider";
+import { FastifyInstance } from "fastify";
+import { IncomingMessage, Server, ServerResponse } from "http";
+import { WorkspaceService } from "./workspaces/api";
+import { WorkspaceServiceImpl } from "./workspaces/services/workspace";
+import { UserExternalLinksServiceImpl } from "./user/services/external_links";
+import { UserNotificationBadgeService } from "./notifications/services/bages";
+import {
+  UserNotificationBadgeServiceAPI,
+  UserNotificationPreferencesAPI,
+} from "./notifications/api";
+import { DatabaseServiceAPI } from "../core/platform/services/database/api";
+import { NotificationPreferencesService } from "./notifications/services/preferences";
+import {
+  MessageThreadMessagesServiceAPI,
+  MessageThreadsServiceAPI,
+  MessageUserBookmarksServiceAPI,
+  MessageViewsServiceAPI,
+} from "./messages/api";
+import { ThreadMessagesService } from "./messages/services/messages";
+import { ThreadsService } from "./messages/services/threads";
+import { UserBookmarksService } from "./messages/services/user-bookmarks";
+import { SearchServiceAPI } from "../core/platform/services/search/api";
+import StorageAPI from "../core/platform/services/storage/provider";
+import { PubsubServiceAPI } from "../core/platform/services/pubsub/api";
+import { CounterAPI } from "../core/platform/services/counter/types";
+import { UserServiceImpl } from "./user/services/users/service";
+import { CompanyApplicationServiceImpl } from "./applications/services/company-applications";
+import { ApplicationServiceImpl } from "./applications/services/applications";
+import { ViewsServiceImpl } from "./messages/services/views";
+import { MessagesEngine } from "./messages/services/engine";
+import { FileServiceAPI } from "./files/api";
+import { FileServiceImpl } from "./files/services";
+import { ChannelServiceImpl } from "./channels/services/channel/service";
+import {
+  ChannelPendingEmailService,
+  ChannelService,
+  MemberService,
+  TabService,
+} from "./channels/provider";
+import { MemberServiceImpl } from "./channels/services/member/service";
+import ChannelPendingEmailServiceImpl from "./channels/services/channel/pending-emails/service";
+import { TabServiceImpl } from "./channels/services/tab";
+import AuthServiceAPI from "../core/platform/services/auth/provider";
+import { ConsoleServiceImpl } from "./console/service";
+import { StatisticsServiceImpl } from "./statistics/service";
+import { logger, TwakeServiceProvider } from "../core/platform/framework";
+import assert from "assert";
+
+type PlatformServices = {
+  realtime: RealtimeServiceAPI;
+  auth: AuthServiceAPI;
+  search: SearchServiceAPI;
+  storage: StorageAPI;
+  pubsub: PubsubServiceAPI;
+  counter: CounterAPI;
+};
+
+type CoreServices = {
```

Comment:
The name would be more like "TwakeServices" as core is the name of the folder in which "PlatformServices" are found.